### DISCUSSION TITLE
音质与qq cookie相关

### DISCUSSION THREAD

**Discussion by mlsqq** [1715173329/UnblockNeteaseMusic#1125] [Q&A]
无损音质获取开启本地vip是默认开启的吗？qq音乐源设置了cookie但是添加到我喜欢后双击播放显示加载失败

**Comment by 1715173329** [comment]
默认不启用无损音质获取，需手动指定环境变量。

**Reply by mlsqq** [reply]
qq cookie有绿钻，我添加音乐到我喜欢然后播放就会失败，歌曲 外婆说 - 程佳佳
求大佬测试下

**Reply by 1715173329** [reply]
网易云音乐里面似乎没有这首歌的信息了，我不知道你怎么添加的。
发个 music id 看看？

**Reply by mlsqq** [reply]
就是我先在果核音乐下载下来然后本地里面添加到我喜欢再删除
![image](https://github.com/UnblockNeteaseMusic/server/assets/85686342/41395037-c3ca-4e0e-bd21-097ddb40a4a2)

**Reply by 1715173329** [reply]
那肯定不行啊，你这是本地的记录，文件删了就没了

**Reply by mlsqq** [reply]
有办法播放qq音乐有的歌但是网易没有的嘛

**Reply by 1715173329** [reply]
只能下载到本地听了，目前我们不支持 hack 网易云音乐的曲库信息。

**Reply by mlsqq** [reply]
我通过我上面的方法成功了又，这有个玄学问题，概率不知道什么原因
